activiti-steps/activiti-steps-assertions-api/src/main/java/org/activiti/steps/EventProvider.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2019 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.steps;
+
+import java.util.List;
+
+import org.activiti.api.model.shared.event.RuntimeEvent;
+
+public interface EventProvider {
```

Comment:
@erdemedeiros thinking about naming here.. should be call it something like EventSource ? Does it make more sense than provider?

Reply:
@salaboy, totally, I like `EventSource`! I think it represents better the purpose of this interface.

Reply:
👍